packages/app-extensions/src/notifier/components/StyledNotifier.js

```diff
@@ -198,7 +198,7 @@ const StyledNotifier = styled.div`
       }
 
       .rrt-confirm-holder {
-        z-index: 100000000;  // reset: react-redux-toastr (index.scss) - Stacking: Toaster > Modal > Blocker
+        z-index: 99999;  // reset: react-redux-toastr (index.scss) - Stacking: Toaster > Modal > Blocker
```

Comment:
Do we have some kind of definition when to use what number or is 9999 more or less random?

Reply:
The number **9999** is lower than **99999** of .flatpickr-calendar.open class of the flatpickr. So that the dialog of the flatpickr will always be on top.